docs/azure-prices.md

```diff
@@ -0,0 +1,123 @@
+---
+sidebar_position: 8
+---
+Azure Price Configuration
+============
+
+OpenCost needs access to the Microsoft Azure Billing Rate Card API to access accurate pricing data for your Kubernetes resources.
+
+## Creating a Custom Azure role
+
+Start by creating an Azure role definition. Below is an example definition, replace `YOUR_SUBSCRIPTION_ID` with the Subscription ID where your Kubernetes Cluster lives:
```

Comment:
Maybe this should say "the Subscription ID of the account where your cluster lives"?

Reply:
I changed it to "the ID of the subscription containing your Kubernetes cluster."